modal/functions.py

```diff
@@ -803,16 +803,16 @@ def __init__(
         else:
             self._secrets = secrets
 
-        if isinstance(retries, int):
+        if retries is None or retries == 0:
```

Comment:
instead of changing this line, can you change this line: https://github.com/modal-labs/modal-client/blob/main/modal/retries.py#L64

Reply:
I wasn't sure if that would break something on the server (since now we are sending a `RetryPolicy` vs `None` earlier). But it looks like it won't.

Reply:
just seems nicer to put all the validation in one place